pkg/oc/admin/policy/modify_roles.go

```diff
@@ -42,13 +44,31 @@ var (
 	  %[1]s edit -z serviceaccount1`)
 )
 
+var addRoleToUserLongDesc = templates.LongDesc(`
```

Comment:
Maybe point out that the `oc create rolebinding` and related commands give you better direct control over what object is created.

Reply:
no please, let's not confuse the users with too many things

Reply:
stole [some language](https://github.com/openshift/origin/pull/19991/files#diff-86c38a2d4ef568f72f4f25f663a64073R53) from other commands about 'for more information'.
> To learn more, see information about RBAC and policy, or use the 'get' and 'describe' commands on the following resources: 'clusterroles', 'clusterrolebindings', 'roles', 'rolebindings', 'users', 'groups', and 'serviceaccounts'.